app/helpers/validation/height.py

```diff
@@ -24,9 +25,10 @@ def validate_lon_lat(lon, lat):
 
 
 def validate_sr(sr):
-    if sr not in (21781, 2056):
+    if sr not in VALID_SRID:
         abort(
             400,
-            "Please provide a valid number for the spatial reference system model 21781 or 2056"
+            "Please provide a valid number for the spatial reference system model; "
```

Comment:
Guess the `;` (semicolon) should be a `:` (colon) here?

Reply:
Matter of preference or speaking language, in french it would be a `;` but maybe in english `:` is better I'm not that good in speaking languages 😉 , but I can change it if you think it is a better english ?

Reply:
The world will keep turning either way :wink:
Probably a colon, as a list of valid values will follow. But the user will also understand that with a semicolon

Reply:
I'll change it then it is quickly done

Reply:
My english is slowly getting better with yours review 😉  thanks 👍🏼